eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -6,6 +6,19 @@ jobs:
   condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
   strategy:
     matrix:
+      Linux_ARM:
```

Comment:
Can we build samples as well on ARM? I think the customer can benefit from us having samples that build fine on ARM.

Maybe even tests (but I don't think that is absolutely necessary, nice to have).

I am surprised how little time these legs take:
![image](https://user-images.githubusercontent.com/6527137/91921649-da42c780-ec80-11ea-8a75-b239258693f7.png)

Reply:
These legs have no vcpkg dependencies and compile without samples. If we had to build dependencies in vcpkg they would run longer. There is no support in vcpkg for building ARM on Linux in vcpkg at this time: https://github.com/microsoft/vcpkg/tree/master/triplets 

Samples would need to be built for platforms on which they are running and, in the case of ARM we are building without standard libraries (e.g. printf, etc.) so samples do not build in this state.

Reply:
> Samples would need to be built for platforms on which they are running and, in the case of ARM we are building without standard libraries (e.g. printf, etc.) so samples do not build in this state.

That's exactly the type of reasoning I was looking for! That makes sense. Thanks for sharing, Daniel.